lib/zipkin-tracer/config.rb

```diff
@@ -39,6 +38,9 @@ def initialize(app, config_hash)
       if @sampled_as_boolean
         @logger && @logger.warn("Using a boolean in the Sampled header is deprecated. Consider setting sampled_as_boolean to false")
       end
+      # Record the path info on server receive, even if the zipkin headers were present in the incoming request?
```

Comment:
You mean the configuration file would have a list of keys to be added on each span for both the client and the server?